scripts/variantstore/wdl/GvsImportSample.wdl

```diff
@@ -65,14 +68,21 @@ task CheckForDuplicateData {
       String dataset_name
       String sample_name
       String gvs_sample_id
+      Boolean load_pet
+      Boolean load_ref_ranges
+      Boolean load_vet
       String? service_account_json_path
       # runtime
       Int? preemptible_tries
   }
 
   String has_service_account_file = if (defined(service_account_json_path)) then 'true' else 'false'
+  String vet_check = if (load_vet) then "table_name like 'vet_%' OR " else "false"
+  String ref_check = if (load_ref_ranges) then "table_name like 'ref_ranges_%' OR " else "false"
+  String pet_check = if (load_pet) then "table_name like 'pet_%' " else "false"
 
-
+  String table_where_clause = vet_check + ref_check + pet_check
```

Comment:
I'm not following the logic here… if I say loading vet and ref_ranges… won't these "false"es get into the WHERE clause when you just concatenate them?